pennychat/tests/utils.py

```diff
@@ -0,0 +1,42 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from pennychat.models import PennyChat, FollowUp
+from users.models import UserProfile
+
+
+def generate_user():
+    user, created = UserProfile.objects.get_or_create(real_name='Anonymous Profile',
+                                                      defaults={'email': 'anonymous@profile.com'})
```

Comment:
nit pick - could I talk you into standardizing around using this code format
```python
user, created = UserProfile.objects.get_or_create(
    real_name='Anonymous Profile',
    defaults={'email': 'anonymous@profile.com'},
)
```
rather than the above? The rationale is that the arguments are always one more indent than the command that created the arguments. If you back up and look at the indenting in this entire file the arguments are indented much differently for every function here. Another benefit of my recommended format is that it avoids arguments getting piled up at the right side of the screen as seen above.

Reply:
Yes, definitely!

Reply:
Y'all might consider using the black code formatted and verifying that style in CI.

Reply:
Black is worth considering. I like what gofmt has done to golang development. Black would be the same thing to python. Last time I looked at Black, I didn't like some of the defaults and the defaults couldn't be changed - namely the preference for double quotes. Perhaps I should reconsider.